mentoring/answer.py

```diff
@@ -24,39 +21,146 @@
 # Imports ###########################################################
 
 import logging
-
 from lazy import lazy
 
-from xblock.fragment import Fragment
-
-from .light_children import LightChild, Boolean, Scope, String
 from .models import Answer
-from .utils import render_template
+
+from xblock.core import XBlock
+from xblock.fields import Scope, Float, Integer, String
+from xblock.fragment import Fragment
+from xblock.validation import ValidationMessage
+from xblockutils.resources import ResourceLoader
+from xblockutils.studio_editable import StudioEditableXBlockMixin
+from .step import StepMixin
+import uuid
 
 
 # Globals ###########################################################
 
 log = logging.getLogger(__name__)
+loader = ResourceLoader(__name__)
+
 
+# Make '_' a no-op so we can scrape strings
+def _(text):
+    return text
```

Comment:
@bradenmacdonald you also need to scrape the translations and provide a set of translations in .po files. Translations files must live at conf/locale in your djangoapp. Check out how https://github.com/edx/edx-ora2 has their conf/locale directory set up for more detail.